core/core.gradle.kts

```diff
@@ -12,24 +12,23 @@ application {
     mainClassName = "edu.wpi.grip.core.Main"
 }
 
-val os = osdetector.classifier.replace("osx", "macosx").replace("x86_32", "x86")
-val arch = osdetector.arch.replace("x86_64", "x64")
-
 val withCuda = project.hasProperty("cuda") || project.hasProperty("WITH_CUDA")
 
 if (withCuda) {
     version = "$version-cuda"
 }
 
+createNativeConfigurations()
```

Comment:
Where is this method from?

Reply:
[buildSrc/NativeConfigurations.kt](https://github.com/WPIRoboticsProjects/GRIP/blob/master/buildSrc/src/main/kotlin/NativeConfigurations.kt)

Reply:
Why isn't this added as part of a plugin?